package.json

```diff
@@ -33,17 +33,18 @@
     "clean": "rm -rf .nyc_output dist node_modules",
     "clean:dist": "rm -rf dist",
     "docs": "jsdoc -r dist",
-    "format": "prettier --config .prettierrc.json --write .",
-    "format:check": "prettier --config .prettierrc.json --check .",
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix",
+    "format": "prettier --config .prettierrc.json --ignore-path .prettierignore --write .",
+    "format:check": "prettier --config .prettierrc.json --ignore-path .prettierignore --check .",
+    "lint": "eslint --ignore-path .eslintignore .",
+    "lint:fix": "eslint --ignore-path .eslintignore . --fix",
     "prepare": "npm run build",
-    "test": "mocha",
-    "test:coverage": "nyc mocha",
-    "test:watch": "mocha --watch"
+    "test": "mocha --config .mocharc.json",
+    "test:coverage": "nyc mocha --config .mocharc.json",
+    "test:watch": "mocha --config .mocharc.json --watch"
```

Comment:
These changes to the NPM run scripts should probably be in a separate PR. This also helps document why this needs to be changed